backend/api/users/types.py

```diff
@@ -152,6 +154,15 @@ def can_edit_schedule(self) -> bool:
     def is_python_italia_member(self) -> bool:
         return Membership.objects.active().of_user(self.id).exists()
 
+    @strawberry.field
+    def billing_addresses(self, conference: str) -> list[BillingAddress]:
```

Comment:
make sure we only return billing addresses of the organiser of the conference code you are sending